test/common/wpt.js

```diff
@@ -414,24 +480,22 @@ class WPTRunner {
     this.status = new StatusLoader(path);
     this.status.load();
     this.specMap = new Map(
```

Comment:
Do we need a symbol or could we use a `Set` here instead of a `Map`?

Reply:
Could you elaborate?

Reply:
I'll take a stab at it.

Reply:
Done.